README.md

```diff
@@ -16,6 +16,23 @@ you might also like
 
 [Upcase]: https://upcase.com?utm_source=til
 
+## Contributing a TIL
```

Comment:
Also not unreasonable to point the CONTRIBUTING.md file out as "READ THIS TO CONTRIBUTE" in README.md, (assuming any separation makes sense)